### PR TITLE
New build

### PR DESCRIPTION
**Pull request by fractalic**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,38 +1,15 @@
...
-        artifactory_app_password = "password"
-        artifactory_app_key = "app_key"
+        // define these variables in your personal ~/.gradle/gradle.properties
+        // for example:
+        /*
+            rightmesh_artifactory_username=<username>
+            rightmesh_artifactory_password=<hash of your developer portal password>
+            rightmesh_superpeer_key=<some key that you have access to>
+        */
+        artifactory_app_username = "${rightmesh_artifactory_username}"
```
Comment (conbrad) [inline build.gradle:10]: Does this work with `artifactory_app_{username, password, key}` also?

Comment (fractalic) [reply build.gradle:10]: Yes that should be fine. I think credentials can still sometimes be a pain to manage when working with artifactory-internal, so there's room for improvement on that front here. Just wanted to get this up to clean a lot of unused stuff out of the buildscript.

**Review by conbrad — APPROVED** [review]

**Comment by FrazerSeymour** [comment]
Are we able to switch to the new artifactory server? (Maybe out of scope of this MR.)

**Comment by fractalic** [comment]
> Are we able to switch to the new artifactory server? (Maybe out of scope of this MR.)

Might as well do it now. I can make that change later this afternoon.

**Review by FrazerSeymour — APPROVED** [review]

**Comment by fractalic** [comment]
For unknown reasons I'm not able to compile when using 0.9.0 from artifactory.rightmesh.io/artifactory/maven. I'm going to leave the repo unchanged for now.

**Pull request merged**